File: plugins/markdown/src/org/intellij/plugins/markdown/lang/psi/impl/MarkdownLinkDestinationImpl.java
package org.intellij.plugins.markdown.lang.psi.impl;

import com.intellij.lang.ASTNode;
import org.jetbrains.annotations.NotNull;

/**
 * @deprecated Please use {@link MarkdownLinkDestination} instead.
 */
@Deprecated
public class MarkdownLinkDestinationImpl extends MarkdownLinkDestination {
  public MarkdownLinkDestinationImpl(@NotNull ASTNode node) {
    super(node);
  }
}

File: plugins/markdown/core/src/org/intellij/plugins/markdown/lang/psi/impl/MarkdownLinkDestinationImpl.kt
package org.intellij.plugins.markdown.lang.psi.impl

import com.intellij.extapi.psi.ASTWrapperPsiElement
import com.intellij.lang.ASTNode
import org.intellij.plugins.markdown.lang.psi.MarkdownPsiElement
import org.jetbrains.annotations.ApiStatus

@Deprecated("Please use {@link MarkdownLinkDestination} instead.", ReplaceWith("MarkdownLinkDestination"))
@ApiStatus.ScheduledForRemoval
abstract class MarkdownLinkDestinationImpl(node: ASTNode): ASTWrapperPsiElement(node), MarkdownPsiElement
